src/main/java/com/epam/machine/repository/AdminRepositoryImpl.java

```diff
@@ -95,4 +95,25 @@ public int getByLogin(String login) throws ClassNotFoundException, SQLException
         return id;
     }
 
+    @Override
+    public boolean checkLogin(String login, String password) throws ClassNotFoundException, SQLException {
+        Class.forName(JDBC_DRIVER);
+        ResultSet resultSet = null;
+        int id = 0;
+        try (Connection connection = DriverManager.getConnection(DATA_BASE_URL, ADMIN, PASSWORD);
+             Statement statement = connection.createStatement()) {
+            String sql = "SELECT * FROM administrator";
```

Comment:
why do we need to fetch all rows from administrator table? we can simply use WHERE and AND conditions with login and password, like `select * from administrator where login = login and password = password` and then simply `return resultSet.next()`

Reply:
```java
try (Connection connection = DriverManager.getConnection(DATA_BASE_URL, ADMIN, PASSWORD);
            PreparedStatement statement = connection.prepareStatement("SELECT * FROM administrator where login = ? and password = ?")) {
            statement.setString(1, login);
            statement.setString(2, password);
            resultSet = statement.executeQuery();
            return resultSet.next();
        }
```

Reply:
smth like that